src/monitor/pgautofailover--1.5--1.6.sql

```diff
@@ -554,26 +554,27 @@ CREATE FUNCTION pgautofailover.current_state
    OUT replication_quorum	bool,
    OUT reported_tli         int,
    OUT reported_lsn         pg_lsn,
-   OUT health               integer
+   OUT health               integer,
+   OUT nodecluster          text
  )
 RETURNS SETOF record LANGUAGE SQL STRICT
 AS $$
    select kind, nodename, nodehost, nodeport, groupid, nodeid,
           reportedstate, goalstate,
    		  candidatepriority, replicationquorum,
-          reportedtli, reportedlsn, health
+          reportedtli, reportedlsn, health, nodecluster
      from pgautofailover.node
      join pgautofailover.formation using(formationid)
     where formationid = formation_id
  order by groupid, nodeid;
 $$;
 
-grant execute on function pgautofailover.current_state(text)
-   to autoctl_node;
-
 comment on function pgautofailover.current_state(text)
         is 'get the current state of both nodes of a formation';
 
+grant execute on function pgautofailover.current_state(text)
+   to autoctl_node;
+
```

Comment:
again this grant is only in the upgrade, not in the `pgautofailover.sql`. I think it might be because they are pure SQL functions.

Reply:
In the SQL extension script, from this function on, GRANT statements are missing. It would be cleaner to go a special PR for fixing that, I believe. Merged as-is, then.